core/lib/src/rocket.rs

```diff
@@ -66,40 +67,38 @@ pub(crate) struct Token;
 impl Rocket {
     #[inline]
     fn _mount(&mut self, base: Origin<'static>, routes: Vec<Route>) {
-        info!("{}{} {}{}",
-              Paint::emoji("🛰  "),
-              Paint::magenta("Mounting"),
-              Paint::blue(&base),
-              Paint::magenta(":"));
+        let span = tracing::info_span!("mounting", at = %Paint::blue(&base), "{} Mounting", Paint::emoji("🛰 "),);
+        let _e = span.enter();
 
         for route in routes {
             let old_route = route.clone();
             let route = route.map_base(|old| format!("{}{}", base, old))
                 .unwrap_or_else(|e| {
-                    error_!("Route `{}` has a malformed URI.", old_route);
-                    error_!("{}", e);
+                    let span = tracing::error_span!("malformed_uri", "Route `{}` has a malformed URI.", old_route);
+                    error!(parent: &span, "{}", e);
                     panic!("Invalid route URI.");
                 });
 
-            info_!("{}", route);
+            tracing::info!(%route);
```

Comment:
Why `tracing::info!` here and `info!` elsewhere in the same file?

Reply:
my bad — i think that's just because i edited this file on multiple different days and may have forgotten that there was an import for the macros. i'll change this to be more consistent, thanks for catching that!